src/components/Tool/Appeal/Modals/AddExcludedContactModal/AddExcludedContactModal.test.tsx

```diff
@@ -0,0 +1,209 @@
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SnackbarProvider } from 'notistack';
+import { I18nextProvider } from 'react-i18next';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { AppealsWrapper } from 'pages/accountLists/[accountListId]/tools/appeals/AppealsWrapper';
+import i18n from 'src/lib/i18n';
+import theme from 'src/theme';
+import { AppealsContext } from '../../AppealsContext/AppealsContext';
+import { AppealQuery } from '../AddContactToAppealModal/appealInfo.generated';
+import { AddExcludedContactModal } from './AddExcludedContactModal';
+
+const accountListId = 'abc';
+const appealId = 'appealId';
+const contactId = 'contact-3';
+const bulkContactIds = [contactId, 'contact-4', 'contact-5'];
+const router = {
+  query: { accountListId },
+  isReady: true,
+};
+const handleClose = jest.fn();
+const mutationSpy = jest.fn();
+const refetch = jest.fn();
+
+const mockEnqueue = jest.fn();
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+const appealMock: AppealQuery = {
+  appeal: {
+    id: appealId,
+    contactIds: ['contact-1', 'contact-2'],
+  },
+};
+
+const Components = ({
+  contactIds = [contactId],
+}: {
+  contactIds?: string[];
+}) => (
+  <I18nextProvider i18n={i18n}>
+    <LocalizationProvider dateAdapter={AdapterLuxon}>
+      <SnackbarProvider>
+        <ThemeProvider theme={theme}>
+          <TestRouter router={router}>
+            <GqlMockedProvider<{
+              Appeal: AppealQuery;
+            }>
+              mocks={{
+                Appeal: appealMock,
+              }}
+              onCall={mutationSpy}
+            >
+              <AppealsWrapper>
+                <AppealsContext.Provider
+                  value={{
+                    accountListId,
+                    appealId: appealId,
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
+                    contactsQueryResult: { refetch },
+                  }}
+                >
+                  <AddExcludedContactModal
+                    handleClose={handleClose}
+                    contactIds={contactIds}
+                  />
+                </AppealsContext.Provider>
+              </AppealsWrapper>
+            </GqlMockedProvider>
+          </TestRouter>
+        </ThemeProvider>
+      </SnackbarProvider>
+    </LocalizationProvider>
+  </I18nextProvider>
+);
+
+describe('AddExcludedContactModal', () => {
+  beforeEach(() => {
```

Comment:
I believe that mocks are reset before each test automatically.

Reply:
Really! I didn't know that

Reply:
You're correct!